motionMonitor/motionMonitor.c

```diff
@@ -1,66 +1,93 @@
 #include "interfaces.h"
 #include "legato.h"
 #include "util.h"
+#include <pthread.h>
 
-static const char FormatStr[] =
-    "/sys/bus/i2c/devices/3-0068/iio:device0/in_%s_%s";
-static const char AccType[] = "accel";
-static const char GyroType[] = "anglvel";
-static const char CompX[] = "x_raw";
-static const char CompY[] = "y_raw";
-static const char CompZ[] = "z_raw";
+void *impactMonitor(void *);
+
+static const char FormatStr[] = "/sys/devices/i2c-0/0-0068/iio:device0/in_%s_%s";
+static const char AccType[]   = "accel";
+static const char GyroType[]  = "anglvel";
+static const char CompX[]     = "x_raw";
+static const char CompY[]     = "y_raw";
+static const char CompZ[]     = "z_raw";
 static const char CompScale[] = "scale";
+bool hasSuddenImpact          = false;
+static const int  impactThreshold = 20;
 
 typedef struct {
   double x;
   double y;
   double z;
 } Acceleration;
 
-bool hasSuddenImpact = false;
+pthread_t impact_thread;
```

Comment:
This is a total nitpick and my fault for not explaining naming conventions (I'll add a note to a readme for future employees and reference). I've been trying to use `camelCase` for variables and `brnkl_camelCase_camelCase` for interprocess functions, even though Legato uses `le_camelCase_PascalCase` for interprocess functions, and `PascalCase` for global variables (that's where `CompX` instead of `compX` came from).

I'll fix the `PascalCase` globals at top in a future commit.